docs/security.rst

```diff
@@ -142,6 +142,32 @@ Now you can continue using urllib3 as you normally would.
 
 For more details, check the :mod:`~urllib3.contrib.pyopenssl` module.
 
+Installing urllib3 with SNI support and certificates
```

Comment:
We need a kickstarter for rewriting the urllib3 docs. :P Good enough for now (tm).